tests/ui/test_discopane.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from pages.discopane import DiscoveryPane
+
+@pytest.fixture(scope='session')
+def base_url(base_url):
+    # This is hardcoded for now, once we go through about:addons, we'll be able to use the template expansion
+    if base_url[len(base_url)-1] != '/':
+        base_url += '/'
+    base_url += 'en-US/firefox/discovery/pane/48.0/Darwin/normal'
+    if 'http' not in base_url:
+        base_url = 'https://' + base_url
+    return base_url
+
+@pytest.fixture
+def firefox_profile(firefox_profile, base_url):
+    '''
+    Set preferences necessary to load discovery pane, and
+    for installing extensions and toggling switches to
+    install and uninstall.
+    '''
+    firefox_profile.set_preference('extensions.webapi.testing', True)
+    firefox_profile.set_preference('extensions.webservice.discoverURL', base_url)
+    firefox_profile.update_preferences()
+    return firefox_profile
+
+@pytest.fixture
+def discovery_pane(selenium, base_url):
+    return DiscoveryPane(selenium, base_url).open()
+
+@pytest.mark.nondestructive
+def test_that_discovery_pane_loads(discovery_pane):
+    assert discovery_pane.is_discopane_visible
+    assert (len(discovery_pane.uninstalled_addons) == 7)
+
+@pytest.mark.nondestructive
+def test_that_welcome_video_plays(discovery_pane):
+    discovery_pane.play_video()
+    assert discovery_pane.is_video_playing
+    discovery_pane.close_video()
+    assert discovery_pane.is_video_closed
```

Comment:
Please also check that the video is not playing before we click it, and the video is not closed before we close it. This reduces the risk of false positives, and gives more confidence for our asserts.